docs/testing/testnet.md

```diff
@@ -0,0 +1,8 @@
+run these SEPERATELY to avoid {possible?} collisions
+
+cargo nextest run --run-ignored=all orchard_glory_goddess
+cargo nextest run --run-ignored=all sapling_glory_goddess
+cargo nextest run --run-ignored=all shield_glory_goddess
+
+to update the testnet wallet run
```

Comment:
@nachog00 can you please take a look at how we're stashing this test data, and check if there's a convenient pattern that can stash it outside the repo (and keep it automatically updated)?